recipes/oatpp/all/test_package/example.cpp

```diff
@@ -0,0 +1,31 @@
+#include "UrlTest.hpp"
+
+#include "oatpp/core/concurrency/SpinLock.hpp"
+#include "oatpp/core/base/Environment.hpp"
+
+#include <iostream>
+#include <atomic>
+
+
+void runTests() {
+    OATPP_RUN_TEST(oatpp::test::network::UrlTest);
```

Comment:
Hey @frek818 ,

I recommend adding the following two tests (as they can fail on a `shared` lib build):
- https://github.com/oatpp/oatpp/blob/master/test/oatpp/core/data/mapping/type/TypeTest.hpp
- https://github.com/oatpp/oatpp/blob/master/test/oatpp/parser/json/mapping/DeserializerTest.hpp

Reply:
@uilianries and @lganzzzo  or Is the best practice to create a separate package for the extension modules (e.g. oatpp-websocket, oatpp-swagger) and require oatpp/1.0.0 for each extension?

I start working on a PR for oatpp-websocket (https://github.com/conan-io/conan-center-index/pull/1689). I think an alternative option would be to add options to the oatpp recipe that would embed the extension in this packge.